cdk/eks/package.json

```diff
@@ -25,7 +25,7 @@
     "@aws-cdk/lambda-layer-kubectl-v31": "^2.0.0"
   },
   "dependencies": {
-    "aws-cdk-lib": "2.179.0",
+    "aws-cdk-lib": "2.186.0",
```

Comment:
Is `2.186.0` the min required version? If that's the case, you need to update the prerequisites section in readme: https://github.com/aws/amazon-cloudwatch-agent/blob/08fcbeb4ef5cc47c70294bb5f1ee9bf41699519c/translator/tocwconfig/sampleConfig/appsignals_fallback_and_eks_config.json#L34